# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libjar/test/unit/test_bug1328865.js
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c modules/libjar/test/unit/test_bug1328865.js
--- a/modules/libjar/test/unit/test_bug1328865.js	Tue Feb 12 17:58:19 2019 +0100
+++ b/modules/libjar/test/unit/test_bug1328865.js	Tue Feb 12 16:08:25 2019 +0000
@@ -15,7 +15,7 @@
   var outerJarBase = "jar:" + ios.newFileURI(file).spec + "!/";
   var goodSpec = "jar:" + outerJarBase + "inner.jar!/hello#!/ignore%20this%20part";
   var goodChannel = NetUtil.newChannel({uri: goodSpec, loadUsingSystemPrincipal: true});
-  var instr = goodChannel.open2();
+  var instr = goodChannel.open();
 
   ok(!!instr, "Should be able to open channel");
 });
@@ -27,7 +27,7 @@
   var outerJarBase = "jar:" + ios.newFileURI(file).spec + "!/";
   var goodSpec = "jar:" + outerJarBase + "inner.jar!/hello?ignore%20this%20part!/";
   var goodChannel = NetUtil.newChannel({uri: goodSpec, loadUsingSystemPrincipal: true});
-  var instr = goodChannel.open2();
+  var instr = goodChannel.open();
 
   ok(!!instr, "Should be able to open channel");
 });
@@ -39,7 +39,7 @@
   var outerJarBase = "jar:" + ios.newFileURI(file).spec + "!/";
   var goodSpec = "jar:" + outerJarBase + "inner.jar!/hello?ignore#this!/part";
   var goodChannel = NetUtil.newChannel({uri: goodSpec, loadUsingSystemPrincipal: true});
-  var instr = goodChannel.open2();
+  var instr = goodChannel.open();
 
   ok(!!instr, "Should be able to open channel");
 });